ROMFS/px4fmu_common/init.d/rc.gnd_apps

```diff
@@ -0,0 +1,24 @@
+#!nsh
+#
+# Standard apps for fixed wing
+#
+
+#
+# Start the attitude and position estimator
+#
+ekf2 start
+#attitude_estimator_q start
+#local_position_estimator start
+
+
+#
+# Start attitude controller
+#
+gnd_att_control start
+gnd_pos_control start
+
+
+#
+# Start Land Detector
+#
+land_detector start rover
```

Comment:
If vehicle type is `ugv`, then it's not `rover`. But I suppose this is to be removed soon also.